### PR TITLE
CRM-19491: Set up email preview

### PR DESCRIPTION
**Pull request by monishdeb**
Cleanup of https://github.com/civicrm/civicrm-core/pull/6564

---

 * [CRM-19491: Set up email preview](https://issues.civicrm.org/jira/browse/CRM-19491)

### CODE REVIEW

**Comment by totten** [comment]
Note: there's a syntax error in `settings/Mailing.setting.php`, and `phpcs` goes unusually bonkers on it. I'd suggest fixing that before posting another update here.

**Comment by monishdeb** [comment]
@totten yup I have fixed that earlier and ```squash```ed with https://github.com/civicrm/civicrm-core/pull/9719/commits/383760d6e3a5b7494c8db68be9f96799e2de40db

File: api/v3/Prevem.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ +--------------------------------------------------------------------+
+ | CiviCRM version 4.6                                                |
```
Comment (seamuslee001) [inline api/v3/Prevem.php]: @monishdeb should this be updated and likewise the Copyright

File: Civi/Angular/Manager.php [hunk 2]
```diff
@@ -62,7 +62,7 @@ public function getModules() {
       $angularModules = array();
       $angularModules['angularFileUpload'] = array(
         'ext' => 'civicrm',
-        'js' => array('bower_components/angular-file-upload/angular-file-upload.min.js'),
+        'js' => array('bower_components/angular-file-upload/dist/angular-file-upload.min.js'),
```
Comment (seamuslee001) [inline Civi/Angular/Manager.php]: @monishdeb  is this right?

File: CRM/Utils/URL.php [hunk 3]
```diff
@@ -0,0 +1,53 @@
+<?php
+
```
Comment (seamuslee001) [inline CRM/Utils/URL.php]: This needs the header added

**Comment by monishdeb** [comment]
@seamuslee001 yup currently the PR is not final and need some additional fix. For now, the PR is just a cleanup of #6564, reason why I marked it with ```wip```. Thanks for your feedback, will update the PR soon with desired changes :)

**Comment by eileenmcnaughton** [comment]
@monishdeb this is something you were working on pre JMA & I don't think there is active work on it or a plan to progress it.

What about you rebase against master & squash into one commit & then we close the PR, link to the commit from JIRA & track there?

**Pull request closed without merging**